bftengine/include/bftengine/Replica.hpp

```diff
@@ -25,7 +25,12 @@
 namespace bftEngine {
 
 // Possible values for 'flags' parameter
-enum MsgFlag : uint8_t { EMPTY_FLAGS = 0x0, READ_ONLY_FLAG = 0x1, PRE_EXECUTE_FLAG = 0x2, PRE_EXECUTED_FLAG = 0x4 };
+enum MsgFlag : uint8_t {
+  EMPTY_FLAGS = 0x0,
+  READ_ONLY_FLAG = 0x1,
+  PRE_PROCESS_FLAG = 0x2,
+  HAS_PRE_PROCESSED_FLAG = 0x4
```

Comment:
Why are we renaming those? For me, the preprocessor is a component (an implementation detail) of pre-execution. 

IMO those flags should continue to be called PRE_EXECUTE*.

Reply:
Guy proposed to give a more generic name to the code related to the pre-execution feature and I agreed.